File: code/ex58.py
class Solution:
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        
        split = s.split(' ')

        while len(split) > 0:
            if split[-1] == '':
                split.pop(-1)
            else:
                break

        if len(split) == 0:
            return 0
        else:
            return len(split[-1])

sol = Solution()
print(sol.lengthOfLastWord(" a"))

File: code/ex58.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        s += ' ';
        int temp = 0, len = 0;
        for (char c : s) {
            if (c == ' ') {
                if (temp > 0)    len = temp;
                temp = 0;
            } else {
                temp++;
            }
        }
        return len;
    }
};